src/stache/includes/cli/template.html

```diff
@@ -17,3 +17,7 @@
     <sky-code>-t &lt;ANY-GIT-URL&gt;</sky-code> — Allows for complete flexibility by cloning any valid Git URL.
   </li>
 </ul>
+
+<sky-alert>
+  Please note a versioned branch name using the <sky-code>[MAJOR_VERSION].x.x</sky-code> format is required to exist in the template git repository. At the time of writing, the <sky-code>4.x.x</sky-code> branch must exist.
```

Comment:
- I'd delete `Please note`. Our style is to generally avoid 'please', and we formatted this as an alert, so I think that implicitly tells users to `note` the content.
- I'd change `using` to either `with` or `that uses`.
- I'd delete `to exist`. I think the meaning here is same without it.
- Should `template` be `template's`? It seems like.
- I'd change `repository` be `repo`. (I know they are the same thing, but we generally go with `repo`.)
- What do you think of changing `At the time of writing, the...` to `For example, as of SKY UX version 4.0, the...`?